webrender/src/platform/unix/font.rs

```diff
@@ -146,40 +146,41 @@ impl FontContext {
                 FontRenderMode::Subpixel => FT_Render_Mode::FT_RENDER_MODE_LCD,
             };
 
-            unsafe {
-                let result = FT_Render_Glyph(slot, render_mode);
-
-                if result.succeeded() {
-                    let bitmap = &(*slot).bitmap;
-
-                    let metrics = &(*slot).metrics;
-                    let mut glyph_width = (metrics.width >> 6) as i32;
-                    let glyph_height = (metrics.height >> 6) as i32;
-                    let mut final_buffer = Vec::with_capacity(glyph_width as usize *
-                                                              glyph_height as usize *
-                                                              4);
-
-                    if bitmap.pixel_mode == FT_Pixel_Mode::FT_PIXEL_MODE_MONO as u8 {
-                        // This is not exactly efficient... but it's only used by the
-                        // reftest pass when we have AA disabled on glyphs.
-                        let offset_x = (metrics.horiBearingX >> 6) as i32 - (*slot).bitmap_left;
-                        let offset_y = (metrics.horiBearingY >> 6) as i32 - (*slot).bitmap_top;
-
-                        // Due to AA being disabled, the bitmap produced for mono
-                        // glyphs is often smaller than the reported glyph dimensions.
-                        // To account for this, place the rendered glyph within the
-                        // box of the glyph dimensions, filling in invalid pixels with
-                        // zero alpha.
-                        for iy in 0..glyph_height {
-                            let y = iy - offset_y;
-                            for ix in 0..glyph_width {
-                                let x = ix + offset_x;
-                                let valid_byte = x >= 0 &&
-                                                 y >= 0 &&
-                                                 x < bitmap.width as i32 &&
-                                                 y < bitmap.rows as i32;
-                                let byte_value = if valid_byte {
-                                    let byte_index = (y * bitmap.pitch as i32) + (x >> 3);
+            let result = unsafe { FT_Render_Glyph(slot, render_mode) };
+
+            if result.succeeded() {
+                let bitmap = unsafe { &(*slot).bitmap };
+
+                let metrics = unsafe { &(*slot).metrics };
+                let mut glyph_width = (metrics.width >> 6) as i32;
+                let glyph_height = (metrics.height >> 6) as i32;
+                let mut final_buffer = Vec::with_capacity(glyph_width as usize *
```

Comment:
Do mean have these lines lined up like this?

`...(*slot).metrics`
`...(metrics.width >> 6)`
`...(metrics.height >> 6)`